### PR TITLE
Refactor ContentLog class. Use FloatField instead of IntegerField for progress field.

### PR DESCRIPTION
**Pull request by djallado**
@aronasorman.

Branch:  develop
Issue: https://github.com/learningequality/ka-lite/issues/2767
Related PR: https://github.com/learningequality/ka-lite/pull/2790

I changed `IntegerField` to `FloatField` for percentage use. 

Attached screen shoot:
Sample

### CODE REVIEW

**Comment by aronasorman** [comment]
@rtibbles any other parts of the code affected by this?

**Comment by cpauya** [comment]
Yikes, I closed the #2767 issue and this PR got closed too.  

@djallado Can you please open another issue re use of FloatField and point this PR on that instead?

**Comment by cpauya** [comment]
Ok, found the `reopen branch` button and will reopen this PR now.  Sorry about that.

File: kalite/distributed/static/js/distributed/content/views.js [hunk 1]
```diff
@@ -136,6 +136,10 @@ window.ContentBaseView = BaseView.extend({
         this.log_model.set({
             points: Math.min(this.possible_points, Math.floor(this.possible_points * progress))
         });
+
```
Comment (rtibbles) [inline kalite/distributed/static/js/distributed/content/views.js]: No need to trigger two change events here, can pass the progress: progress key: value pair in the previous .set method call.

File: kalite/main/models.py [hunk 2]
```diff
@@ -508,11 +508,10 @@ class ContentLog(DeferredCountSyncedModel):
     progress_timestamp = models.DateTimeField(blank=True, null=True)
     content_source = models.CharField(max_length=100, db_index=True)
     content_kind = models.CharField(max_length=100, db_index=True)
-    progress = models.IntegerField(blank=True, null=True) # TODO(djallado): Change this to Floatfield for percentage.
+    progress = models.FloatField(blank=True, null=True)  # (djallado): Changed this to Floatfield for percentage.
```
Comment (rtibbles) [inline kalite/main/models.py]: Remove TODO :)

**Comment by rtibbles** [comment]
Couple of comments.

@aronasorman can't think of anything else that would be affected by this. Mostly just amazed that this didn't totally blow up before without this!

**Comment by djallado** [comment]
@aronasorman. I am replicating this with student account and I view a pdf. I noticed that `complete` field is always return true even if I didn't scroll the pdf.

**Comment by djallado** [comment]
I add `if` condition in `set_complete` method of `ContentLogModel` so that I can assure that the `complete` field will return true if `progress` is 1

**Pull request merged**